### ISSUE TITLE
The Dutch channel BVN

### ISSUE THREAD

**Issue by ErikBrown2** [retrospect-addon/plugin.video.retrospect#1532]
### What Channel would you like to add and why?

The Dutch channel BVN https://www.bvn.tv/ . The programs are similar to what is shown on NPO but it happens regularly that interesting programs are geo blocked on NPO but are available on BVN. So the channel will be very interesting for Dutch people living outside of Europe.

### Provide some details for the channel
- Name of the channel in English: BVN
- Name of the channel in native language: BVN
- Website for the channel: https://www.bvn.tv/
- If available link to the Android app for the VOD content of channel: https://play.google.com/store/apps/details?id=tv.bvn.app
- Link to the logo of the channel https://www.bvn.tv/wp-content/themes/bvn/assets/img/bvn-logo.png
 
### Check which features should be implemented
- [X] Video on-demand content
- [X] TV live streams
- [ ] Use your account for the VOD services from Retrospect
- [ ] Access the premium content if you are eligible

**Comment by basrieter** [comment]
Duplicate of #1507
